### PR TITLE
[MERL-747] Bug: Fix HMR for wp-templates

### PR DESCRIPTION
**Pull request by theodesp**
## Tasks

- [x] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.

## Description

Fixes issue with HMR not working with wp-templates

## Testing
1. Start the example project in dev mode.

2. Edit the content of the template in wp-templates

3. The page should refresh so the developer can see the updates on the fly.

### CODE REVIEW

File: packages/faustwp-core/src/components/WordPressTemplate.tsx [hunk 1]
```diff
@@ -21,13 +21,6 @@ export type FaustTemplateProps<Data, Props = Record<string, never>> = Props & {
   __TEMPLATE_VARIABLES__?: { [key: string]: any };
 };
 
-function cleanTemplate(
```
Comment (theodesp) [inline packages/faustwp-core/src/components/WordPressTemplate.tsx:24]: This is problematic as HMR does not pick up the React Component when it changes.

File: .changeset/modern-tables-live.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/core': patch
+---
+
+Bug fix: Fixes HMR issue with wp-templates
```
Comment (blakewilson) [inline .changeset/modern-tables-live.md:5]:
```suggestion
Bug fix: Fixes Hot Module Reload (HMR) issue within wp-templates
```

**Review by blakewilson — COMMENTED** [review]
Awesome work @theodesp! Glad we got this fixed. Added a minor suggestion for the changeset just in case people are not familiiar with "HMR"

**Review by josephfusco — APPROVED** [review]
Tested and works as expected! Thank you @theodesp!

**Pull request merged**